### PR TITLE
Update/v1.63.1

### PR DESCRIPTION
**Pull request by k0gen**

### CODE REVIEW

File: scripts/embassy.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
 export { setConfig } from "./services/setConfig.ts";
 export { properties } from "./services/properties.ts";
-export { getConfig } from "./services/getConfig.ts";
\ No newline at end of file
+export { getConfig } from "./services/getConfig.ts";
+export { migration } from "./services/migrations.ts";
```
Comment (chrisguida) [inline scripts/embassy.ts:4]:
```suggestion
export { migration } from "./services/migrations.ts";

```

File: scripts/services/migrations.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import {compat, types as T} from "../deps.ts";
+
+export const migration: T.ExpectedExports.migration = compat.migrations
+    .fromMapping( {}, "1.63.1" );
```
Comment (chrisguida) [inline scripts/services/migrations.ts:4]:
```suggestion
    .fromMapping( {}, "1.63.1" );

```

File: README.md [hunk 3]
```diff
@@ -33,51 +33,53 @@ docker run --privileged --rm linuxkit/binfmt:v0.8
...
 
 ```
 make
 ```
 
 ## Installing (on Embassy)
 
-SSH into an Embassy device.
-`scp` the `.s9pk` to any directory from your local machine.
+Secure copy (`scp`) the `synapse.s9pk` to your Embassy and SSH in.
```
Comment (chrisguida) [inline README.md:78]: This is not necessary when using remote install with `embassy-cli`

Comment (k0gen) [reply README.md:78]: Planing to change it to include embassy-cli and mention the option to side load via web gui

**Pull request merged**